data/data/bootstrap/files/usr/local/bin/report-progress.sh

```diff
@@ -16,22 +14,14 @@ wait_for_existance /opt/openshift/.bootkube.done
 wait_for_existance /opt/openshift/.openshift.done
 
 echo "Reporting install progress..."
-timestamp="$(date -u +'%Y-%m-%dT%H:%M:%SZ')"
 while ! oc --config="$KUBECONFIG" create -f - <<-EOF
 	apiVersion: v1
-	kind: Event
+	kind: ConfigMap
 	metadata:
-	  name: "${NAME}"
-	  namespace: kube-system
-	involvedObject:
-	  namespace: kube-system
-	message: "${MESSAGE}"
-	firstTimestamp: "${timestamp}"
-	lastTimestamp: "${timestamp}"
-	count: 1
-	source:
-	  component: cluster
-	  host: $(hostname)
+	  name: bootstrap
+	  namespace: openshift
```

Comment:
try to keep this in kube-system namespace.

Reply:
3118a6185...1a3cc8be6

Reply:
This is still `openshift` ;)

Reply:
/hold

Until this gets resolved ;).

Reply:
Oops, git fail. Sorry.

1a3cc8be6...4d90daf27

/hold cancel